tests/ert_tests/dark_storage/test_http_endpoints.py

```diff
@@ -100,9 +102,11 @@ def test_get_response(poly_example_tmp_dir, dark_storage_client):
     stream = io.BytesIO(resp.content)
     response_df2 = pd.read_csv(stream, index_col=0, float_precision="round_trip")
 
-    assert (
-        len(response_df1.columns) == 10
-    ), f"\nexperiment_json: {json.dumps(experiment_json, indent=1)} \n\nensemble_json: {json.dumps(ensemble_json, indent=1)}\n status_code: {resp.status_code}"
+    assert len(response_df1.columns) == 10, (
+        f"\nexperiment_json: {json.dumps(experiment_json, indent=1)} \n\n"
+        f"ensemble_json: {json.dumps(ensemble_json, indent=1)}\n"
+        f" status_code: {resp.status_code}"
+    )
```

Comment:
I chose round-bracket-string-concatenation because I found it a tad more readable here. But that might be a matter of taste.

Sorted imports (`isort`) and removed the superfluous `import` in this file. Bonus change..

Reply:
Yes, I already googled that brackets is a preferred way to split the line and not backslash